setup.py

```diff
@@ -2,9 +2,11 @@
 
 from setuptools import setup, find_packages
 
+import edx_oauth2_provider
+
 setup(
     name='edx-oauth2-provider',
-    version='1.0.1',
+    version=edx_oauth2_provider.__version__,
```

Comment:
Any reason you're going back to this model? We tried it with auth-backends, and I thought we didn't like it.